redis/src/commands/cluster_scan.rs

```diff
@@ -487,7 +487,7 @@ where
 async fn send_scan<C>(
     scan_state: &ScanState,
     core: &C,
-    match_pattern: Option<String>,
+    match_pattern: Option<Vec<u8>>,
```

Comment:
`BytesMut` is more efficient, any special reason why you choose `Vec<u8>` ? (`BytesMut` is similar to `Vec<u8>` but with less copies and allocations)

Reply:
The built in function of the library is to_redis_args which return Vec<u8>

Reply:
Might be better to refactor but not at this point

Reply:
Makes sense not to make a drastic change at this point